File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/01. Series Calculator/01. Series Calculator.py
serial_name = input()
seasons_count = int(input())
chapters_count = int(input())
time_for_serial = float(input())

ad_time = time_for_serial * 0.20
all_chapter_time = time_for_serial + ad_time
bonus_time = seasons_count * 10
total_time =  all_chapter_time * chapters_count  * seasons_count+ bonus_time

print(f'Total time needed to watch the {serial_name} series is {total_time:.0f} minutes.')


'''
Game of Thrones
7
10
50
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/01. Series Calculator/01. Series Calculator.js
function seriesCalculator(serialName, seasonsCount, chaptersCount, timeForSerial) {
    let adTime = +timeForSerial * 0.20
    let allChapterTime = +timeForSerial + adTime
    let bonusTime = +seasonsCount * 10
    let totalTime = allChapterTime * +chaptersCount * +seasonsCount + bonusTime

    return `Total time needed to watch the ${serialName} series is ${totalTime.toFixed(0)} minutes.`
}

//console.log(seriesCalculator('Lucifer', 3, 18, 55))